tracer/test/Datadog.Trace.Tests/Telemetry/Metrics/common_metrics.json

```diff
@@ -519,9 +519,7 @@
       "user_tags":[]
     },
     "request.tainted": {
-      "tags": [
-        "vulnerability_type"
-      ],
+      "tags": [],
```

Comment:
what's this file for? it generated or manually maintained. I guess the tag was incorrectly applied before?

Reply:
It is a copy of the file that is defined in https://github.com/DataDog/dd-go and defines the metric values used. It was wrong : this topic was discussed with the Iast team and they agreed in this change. After merging, I will need to create a separate PR in https://github.com/DataDog/dd-go and get an approval from the telemetry team.

Reply:
This file is common with other teams (we copy it, but after that, we need to update the original file in another repository), the metric was already defined, it will need an update after merging.